src/commands/deploy.rs

```diff
@@ -79,20 +79,26 @@ pub struct DeployCommand {
     /// for readiness.
     #[clap(long = "readiness-timeout", default_value = "60")]
     pub readiness_timeout_secs: u16,
+
+    // Deploy to a named target.
+    #[clap(
+        name = "depoloyment-id",
+        long = "deployment-id",
+        env = DEPLOYMENT_ID_ENV
+    )]
+    pub deployment_id: Option<String>,
 }
 
 impl DeployCommand {
     pub async fn run(self) -> Result<()> {
-        let path = dirs::config_dir()
-            .context("Cannot find config directory")?
-            .join("spin")
```

Comment:
Do we want to put this under a `fermyon` directory instead?
There might be other tools (that we build) that might need to share that configuration file.

Reply:
This was just following the existing convention.  I don't have a strong opinion either way.